app/services/filter_service.rb

```diff
@@ -14,6 +14,8 @@ def call
 
   def data_services
     data_services = DataService.includes(:organisation)
+    return data_services if @query.blank? && @filters.blank?
```

Comment:
You don't need this here, since we're already only filtering/searching on the lines below if the params are present 😄